mainwindow.cpp

```diff
@@ -19,11 +19,16 @@
 #include <QClipboard>
 #include <QThread>
 #include <QTextCursor>
-#include <QStandardPaths>
```

Comment:
Shouldn't you have kept this in an ifdef? You might get warnings or issues in certain scenarios.